### DISCUSSION TITLE
blog/apollo-angular-12

### DISCUSSION THREAD

**Discussion by giscus** [the-guild-org/the-guild-website#1003] [Docs Discussions]
# blog/apollo-angular-12

Check what's new in Apollo Angular and how to get the full potential benefits of using Angular + GraphQL + TypeScript combined thanks to GraphQL-Code-Generator

https://the-guild.dev/blog/apollo-angular-12

**Comment by SimonCockx** [comment]
> Next, you run the GraphQL Code Generator — Angular Apollo Plugin to generate types and angular services.

How do you run it? I can't find it documented anywhere.
